freqtrade/commands/cli_options.py

```diff
@@ -82,6 +82,11 @@ def __init__(self, *args, **kwargs):
         help='Reset sample files to their original state.',
         action='store_true',
     ),
+    "recursive_strategy_search": Arg(
+        '-r', '--recursive_strategy_search',
```

Comment:
I'd not give this command a "1 letter alias" 

if you rely on this, you'll usually want to have it in the configuration so it always applies.
For consistency, the command should also be `--recursive-strategy-search` (with dash, not underscore), as all other arguments also use it that way.

Obviously the configuration setting will remain as is.

Reply:
The argument name has not been changed - so it's now the only argument to use `_` in it's name.

Also, did you actually test this with a command line? 

`freqtrade trade  --recursive_strategy_search -s SampleStrategy` yields `freqtrade trade: error: argument --recursive_strategy_search: expected one argument`.